ui/src/engine/pearl/self_serve.rs

```diff
@@ -148,6 +148,18 @@ impl Runner<PEARL_RING_LED_COUNT, PEARL_CENTER_LED_COUNT> {
                             .fade_in(1.5),
                         );
                     }
+                    QrScanSchema::UserV2 => {
+                        self.operator_signup_phase.user_qr_code_ok();
+                        self.set_ring(
+                            LEVEL_FOREGROUND,
+                            animations::Wave::new(
+                                Argb::PEARL_RING_USER_QR_SCAN_WAVE,
+                                5.0,
+                                0.5,
+                                true,
+                            ),
+                        );
```

Comment:
we would need a smooth transition here, see https://github.com/worldcoin/orb-software/pull/401